<?xml version="1.0"?>
<launch>
	<!-- Flip thermal camera  -->
	<node type="thermal_dynamic_detector.py" name="thermal_dynamic_detector" pkg="thermal_dynamic_detector" respawn="true" output="screen">
		<param name="topicImageIn" value="/FlirA65/image_raw" />
		<param name="topicImageOut" value="/det/thermal_dynamic_camera/threshold/heat" />
	</node>
</launch>
